controls/srg_ctr/SRG-APP-000089-CTR-000150.yml

```diff
@@ -2,8 +2,26 @@ controls:
 - id: SRG-APP-000089-CTR-000150
   levels:
   - medium
-  title: {{{ full_name }}} must generate audit records for all DoD-defined auditable
-    events within all components in the platform.
+  title: {{{ full_name }}} must generate audit records for all DoD-defined 
+    auditable events within all components in the platform.
   rules:
   - audit_profile_set
```

Comment:
Hmm, why do you think the rule should be manual? I'm reading the rule `audit_profile_set` and everything in the check and fixtext seems to be handled by that rule well. I think the only thing we should do is to set the value of var_openshift_audit_profile to `WriteRequestBodies`. I /think/ it can be done on the profile level in `srg_ctrl.yaml`, check out how different "global" variables are set in `controls/srg_gpos.yml`.

Reply:
My concerns were around `WriteRequestBodies`. I have set the global variable.